src/index.tsx

```diff
@@ -106,6 +106,14 @@ export class RCPhoneInput extends React.Component<IProps, IState> {
       }
     }
 
+    if (value && this.state.formattedNumber !== value) {
```

Comment:
@eightbits-us This is in `componentDidMount` right?

For which scenario is it? When is mounted with value?

If so, have you tried to find a solution in `mapPropsToState` here https://github.com/join-com/rc-phone-input/blob/master/src/index.tsx#L86

Reply:
It's for saving form entries with redux persist. The problem is that the values from redux is not updating the state so value is not displayed. I'll try to fix it on mapPropsToState

Reply:
@eightbits-us yeah, but from the perspective of this component, it is rendering for the first time with value?